stylize/__main__.py

```diff
@@ -32,10 +33,21 @@ def current_git_commit():
 ## Yields all files that differ from the branching point with @diffbase or are
 # not tracked by git.
 def enumerate_changed_files(exclude=[], diffbase="origin/master"):
-    # find common ancestor between @diffbase and @current_commit
-    ancestor = subprocess.check_output(
-        ['git', 'merge-base', current_git_commit(), diffbase]).strip().decode(
-            'utf-8')
+    try:
+        # try to find common ancestor between @diffbase and @current_commit
+        ancestor = subprocess.check_output(
+            ['git', 'merge-base', current_git_commit(), diffbase]).strip(
+            ).decode('utf-8')
+    except subprocess.CalledProcessError:
+        # There was no common ancestor between @diffbase and @current_commit
+        ancestor = subprocess.check_output(
+            ['git', 'rev-list', '--max-parents=0', current_git_commit()
+             ]).strip().decode('utf-8')
+        print("[ERR] Your diffbase: '" + str(diffbase) +
```

Comment:
It may be best to have this throw an exception here, since it's not really recoverable.  As is, it will fail below this when it finds that `ancestor` is `None`.